README.md

```diff
@@ -0,0 +1,46 @@
+## omsoo-portfolio
+
+The project is still in the works. Constructive feedback is encouraged. Just completed the structure for the destop version and mobile view.
+
+#Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo
+
+[Live Demo Link](https://github.com/omsoo/Skill-Up)
```

Comment:
- [ ] Kindly add a real live demo link in your README or work on deleting it, while keeping broken links in your README is counted as unprofessional